docs/change-existing-WDL-for-Azure.md

```diff
@@ -0,0 +1,59 @@
+# How to modify an existing WDL file to run on Cromwell on Azure
+
+For any pipeline, you can create a [WDL](https://software.broadinstitute.org/wdl/) file that calls your tools in docker containers. Please note that Cromwell on Azure only supports tasks with docker containers defined for security reasons.<br/>
+
+In order to run a WDL file, you must modify/create a workflow with the following runtime attributes for the tasks that are compliant with the [TES or Task Execution Schemas](https://cromwell.readthedocs.io/en/develop/backends/TES/):
+
+```
+runtime {
+    cpu: 1
+    memory: 2 GB
+    disk: 10 GB
+    docker:
+    maxRetries: 0
+}
+```
+Ensure that the attributes `memory` and `disk` (note: use the singular form for `disk` NOT `disks`) have units. Supported units from Cromwell:
+
+> KB - "KB", "K", "KiB", "Ki"<br/>
+> MB - "MB", "M", "MiB", "Mi"<br/>
+> GB - "GB", "G", "GiB", "Gi"<br/>
+> TB - "TB", "T", "TiB", "Ti"<br/>
+
+`preemptible` and `zones` attributes are currently not being passed through Broad's Cromwell to the TES backend, and hence are not supported.<br/>
+Each of these runtime attributes are specific to your workflow and tasks within those workflows. The default values for resource requirements are as set above.<br/>
+Learn more about Cromwell's runtime attributes [here](https://cromwell.readthedocs.io/en/develop/RuntimeAttributes).
+
+## Runtime attributes comparison with a GCP WDL file
+
+Left panel shows a WDL file created for GCP whereas the right panel is the modified WDL that runs on Azure.
+
+
+![Runtime Attributes](/docs/screenshots/runtime.PNG)
+
+
+## Using maxRetries to replace the preemptible attribute
+
+If you want to use the `preemptible` attribute but don’t use `maxRetries` for a task, consider replacing `preemptible` by `maxRetries` to keep the retry functionality. Remember that for Cromwell on Azure, you can either use low-priority VMs in Batch (default configuration) for all tasks or use dedicated VMs for all task by changing the `docker-compose.yml` setting `UsePreemptibleVmsOnly` as described in [this section](/docs/troubleshooting-guide.md/#How-can-I-configure-my-Cromwell-on-Azure-instance-to-use-dedicated-Batch-VMs-to-avoid-getting-preempted?)
```

Comment:
'or use dedicated VMs for all tasks' - plural for task

Reply:
If 'preemptible' is supported, we need to change this part: 1. for us preemptible type is bool, for GCP - int; 2. if you start with GCP wdl and want to keep retries -> you need to use maxRetries instead of preemptible .

Reply:
Changing to suggest using both - preemptible and maxRetries to use the retry with preemptible nodes